Shopify/cli
# Add ui.enable_create option for function configuration schema

### WHAT is this pull request doing?

Adds a new option to the toml for function extensions.

### How to test your changes?
Note that you'll need to have my server-side changes which currently have not been merged yet. For the sake of obscurity, I'm not going to link anything since its a private repo.

* Try to push a function with changing the toml. It should succeed.
* Try to push a function with the change in the toml. It should succeed.
```
[ui]
enable_create = true
```

### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
